gslb/nodes/dq_ingestion.go

```diff
@@ -279,8 +279,8 @@ func AddUpdateObjOperation(key, cname, ns, objType, objName string, wq *utils.Wo
 		return
 	}
 	if metaObj.GetTenant() != gslbutils.GetTenantInNamespaceAnnotation(ns, cname) {
-		gslbutils.Errf("key: %s, msg: %s", key, "tenant mismatch found for the object")
```

Comment:
Should we keep this error message so that we will get an idea which tenant comparision has happend?

Reply:
It should be logged only when we are returning and not processing the key. Currently it is falsely logged as  error even though we choose to process it and not return.